Sources/KituraNIO/HTTP/HTTPServer.swift

```diff
@@ -37,6 +37,8 @@ public class HTTPServer : Server {
 
     private let syncQ = DispatchQueue(label: "HTTPServer.syncQ")
 
+    private var ctx: ChannelHandlerContext?
```

Comment:
@nethraravindran The HTTPServer instance never goes out of scope. Wondering if this would end up holding the related `ByteBuffer` instance forever.